packages/pyroscope-flamegraph/src/Toolbar.tsx

```diff
@@ -68,6 +68,7 @@ export interface ProfileHeaderProps {
   viewDiff: 'diff' | 'total' | 'self';
   handleSearchChange: (s: string) => void;
   highlightQuery: string;
+  ExportData?: ReactNode;
```

Comment:
Since we pass something here it's not really optional right?

Also it may be better to add a default value in this component as opposed to `FlameGraphRenderer`, since this one is closer to the actual usage.